client/my-sites/plan-features-2023-grid/components/storage-add-on-dropdown.tsx

```diff
@@ -22,26 +64,33 @@ export const StorageAddOnDropdown = ( { planSlug, storageOptions }: StorageAddOn
 
 	// TODO: Consider transforming storageOptions outside of this component
 	const selectControlOptions = storageOptions.reduce( ( acc, storageOption ) => {
-		const title = getStorageStringFromFeature( storageOption.slug );
-		if ( title ) {
-			acc.push( {
-				key: storageOption?.slug,
-				name: title,
-			} );
-		}
+		const title = getStorageStringFromFeature( storageOption.slug ) || '';
```

Comment:
Now that there is no longer an `if ( title )` condition, this `reduce` is effectively just a `map`.

Reply:
👍 https://github.com/Automattic/wp-calypso/pull/81338/commits/e9bccbf6824b527997af60272fbd22a37d079010